packages/xo-web/src/xo-app/home/index.js

```diff
@@ -479,8 +478,9 @@ export default class Home extends Component {
 
   state = {
     homeItemsPerPage: +defined(
-      cookies.get('homeItemsPerPage'),
-      DEFAULT_ITEMS_PER_PAGE
+      () => this.props.user.preferences.homeItemsPerPage,
+      () => cookies.get('homeItemsPerPage'),
```

Comment:
Does it make sense to store it both in the cookies and in the preferences? (open discussion and we should probably add the final decision to the issue/PR description)